.github/workflows/buid_and_release.yml

```diff
@@ -39,3 +39,5 @@ jobs:
           output/release/medgen.obo
           output/release/medgen-disease-extract.obo
           output/release/medgen-xrefs.robot.template.tsv
+          output/release/hpo-umls.sssom.tsv
+          output/release/hpo-mesh.sssom.tsv
```

Comment:
## SSSOM Outputs
- [ ] @matentzn @twhetzel review

I ran a [new build](https://github.com/monarch-initiative/medgen/actions/runs/8406868725) and created [a release](https://github.com/monarch-initiative/medgen/releases/tag/2024-03-24) which includes these. Here are some sample rows.

### `hpo-umls.sssom.tsv`
|subject_id|subject_label|predicate_id|object_id|
|-|-|-|-|
|UMLS:C0000727|Acute abdomen|skos:exactMatch|HP:0033400|
|UMLS:C0000729|Abdominal cramps|skos:exactMatch|HP:0032155|
|UMLS:C0000731|Abdominal distention|skos:exactMatch|HP:0003270|

### `hpo-mesh.sssom.tsv`
|subject_id|predicate_id|object_id|
|-|-|-|
|HP:0000003|skos:exactMatch|MESH:D021782|
|HP:0000005|skos:exactMatch|MESH:D040582|
|HP:0000011|skos:exactMatch|MESH:D001750|

This one doesn't have `subject_label` or `object_label` because I couldn't guarantee that the label in `MedGenIDMappings.txt` was an accurate reflection of either source in 100% of cases. I imagine these labels are coming either from UMLS or from MedGen.

### `hpo-mesh-no-matches.sssom.tsv` 
[hpo-mesh-no-matches.sssom.tsv.zip](https://github.com/monarch-initiative/medgen/files/14734397/hpo-mesh-no-matches.sssom.tsv.zip)
|subject_id|predicate_id|object_id|umls_id|umls_label|
|-|-|-|-|-|
| |skos:exactMatch|MESH:C000591739|UMLS:C1970109|Aromatase excess syndrome|
| |skos:exactMatch|MESH:C000596385|UMLS:C3495589|Jalili syndrome|
| |skos:exactMatch|MESH:C000597569|UMLS:C4042185|Teratoid Rhabdoid Tumor|

Not included in the release. This is for analysis. I just wanted you to see the cases where there were no matches. There were only about 2,300 HPO<->MeSH mappings that could be derived out of the ~16,000 MeSH terms that have UMLS mappings. I included `umls_id` and `umls_label` in this set for review. It also includes all of the rows for all of the matches; not just the non-matches. I sorted the non-matches to the top.

Reply:
> Double checking that you 100% want only UMLS mappings

For now yes. Maybe in the future (but not now) we will look for medgen mappings as well, but for now, we only want HPO-UMLS mappings.

> I think I renamed hpo-umls.sssom.tsv incorrectly. Just realizing this now. I have the subjects as UMLS. But HPO comes first in the filename; so perhaps I should make it so that the position is consistent? Does it matter to you which is the subject and which is the object?

Its cosmetic, ask your own inner sense of style. I like the order, but wont insist on it if it takes a lot of time to fix

> I could include mapping_justification, but I don't know their process, and I don't know if we can say if there is or isn't variability in how the do their mappings.

Please use sssom toolkit as you want to release a _valid_ sssom file. `mapping_justification` is mandatory. Since its `MedGen` I would tend to use `semapv:ManualMappingCuration`.

Reply:
I edited your comment w/ checkboxes. Will do.

Reply:
All done!